src/linuxthreads.c

```diff
@@ -316,9 +321,13 @@ static void ListerThread(struct ListerParams *args) {
     sa.sa_sigaction_ = SignalHandler;
     sys_sigfillset(&sa.sa_mask);
     sa.sa_flags      = SA_ONSTACK|SA_SIGINFO|SA_RESETHAND;
+    #if defined (__aarch64__)
+    sys_rt_sigaction(sync_signals[sig], &sa, (struct kernel_sigaction *)NULL, kSigsetSize);
+    #else
     sys_sigaction(sync_signals[sig], &sa, (struct kernel_sigaction *)NULL);
+    #endif
```

Comment:
Is there a reason we can't use rt_sigaction() on intel (that is, just change this unconditionally to use rt_sigaction?)

it's OK if so, I just wonder why.